src/services/helper/entity-type.js

```diff
@@ -0,0 +1,152 @@
+// DependenciesI
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const entityTypeQueries = require('../../database/queries/entityType')
+const { UniqueConstraintError } = require('sequelize')
+
+module.exports = class EntityHelper {
+	/**
+	 * Create entity.
+	 * @method
+	 * @name create
+	 * @param {Object} bodyData - entity body data.
+	 * @param {String} id -  id.
+	 * @returns {JSON} - Entity created response.
+	 */
+
+	static async create(bodyData, id) {
+		bodyData.created_by = '0' || id
+		bodyData.updated_by = '0' || id
+		try {
+			const entityType = await entityTypeQueries.createEntityType(bodyData)
+			return common.successResponse({
+				statusCode: httpStatusCode.created,
+				message: 'ENTITY_CREATED_SUCCESSFULLY',
+				result: entityType,
+			})
+		} catch (error) {
+			if (error instanceof UniqueConstraintError) {
+				return common.failureResponse({
+					message: 'ENTITY_ALREADY_EXISTS',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+			throw error
+		}
+	}
+
+	/**
+	 * Update entity.
+	 * @method
+	 * @name update
+	 * @param {Object} bodyData - entity body data.
+	 * @param {String} _id - entity id.
+	 * @param {String} loggedInUserId - logged in user id.
+	 * @returns {JSON} - Entity updted response.
+	 */
+
+	static async update(bodyData, id, loggedInUserId) {
+		bodyData.updated_by = 1 || loggedInUserId
```

Comment:
[DISCUSS]

Reply:
The temporary solution has been removed.